src/index.html

```diff
@@ -16,17 +16,29 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="page">
+    <form
+      class="form form--big"
+      data-qa="big"
+    >
+      <input
+        type="text"
+        placeholder="Try “Los Angeles“"
+        class="input input--big"
+        data-qa="keypress"
+      />
+    </form>
 
```

Comment:
There should be a blank line between the closing </form> tag of the first form and the opening <form> tag of the second form. According to the checklist, add empty lines between multiline sibling blocks of HTML.